fuzz/fuzz_targets/no-traps.rs

```diff
@@ -0,0 +1,68 @@
+#![no_main]
+
+use arbitrary::Unstructured;
+use libfuzzer_sys::fuzz_target;
+use wasmtime::*;
+
+#[cfg(feature = "wasmtime")]
+#[path = "../../crates/fuzz-stats/src/dummy.rs"]
+pub mod dummy;
+
+// Define a fuzz target that accepts arbitrary
+// `Module`s as input.
+fuzz_target!(|data: &[u8]| {
+    let mut u = Unstructured::new(data);
+    let (wasm_bytes, config) = match wasm_tools_fuzz::generate_valid_module(&mut u, |config, _| {
+        config.disallow_traps = true;
+        config.threads_enabled = false;
+        config.allow_start_export = false;
+        config.max_memories = config.max_memories.min(1);
+        config.memory64_enabled = true;
+        config.exceptions_enabled = false;
+        Ok(())
+    }) {
+        Ok(m) => m,
+        Err(_) => return,
+    };
+    // Validate the module or component and assert that it passes validation.
+    let mut validator = wasmparser::Validator::new_with_features(wasmparser::WasmFeatures {
+        component_model: false,
+        multi_value: config.multi_value_enabled,
+        multi_memory: config.max_memories > 1,
+        bulk_memory: true,
+        reference_types: true,
+        simd: config.simd_enabled,
+        relaxed_simd: config.relaxed_simd_enabled,
+        memory64: config.memory64_enabled,
+        threads: config.threads_enabled,
+        exceptions: config.exceptions_enabled,
+        ..wasmparser::WasmFeatures::default()
+    });
+    if let Err(e) = validator.validate_all(&wasm_bytes) {
+        panic!("Invalid module: {}", e);
+    }
```

Comment:
Might be a little bit easier to read this fuzz target if validation was pulled out into a `validate` function and then calling exports was also pulled out into its own `call_function_exports` function or something like that. As is, it is a bit of a wall of code to read.